apps/frontend/src/lib/storage/types/user/userData/index.ts

```diff
@@ -40,7 +50,7 @@ export const UserDataSchema = z.object({
   lannaHalloween: nullToUndefined(LannaHalloweenDataSchema),
   devcon: nullToUndefined(DevconSchema),
   connectionPSISize: nullToUndefined(ConnectionPSISizeSchema),
```

Comment:
I think this was the wrong place to put the PSI size -- I think it may make more sense to put it with the connection itself.

Reply:
While I like using connection more, I feel like there's enough going on in this PR. I have the commit, but I'll probably make it into a follow up PR.